test/integration/test_groundlight.py

```diff
@@ -336,3 +336,44 @@ def test_submit_numpy_image(gl: Groundlight, detector: Detector):
     assert str(_image_query)
     assert isinstance(_image_query, ImageQuery)
     assert is_valid_display_result(_image_query.result)
+
+
+@pytest.mark.skipif(MISSING_NUMPY or MISSING_PIL, reason="Needs numpy and pillow")  # type: ignore
```

Comment:
numpy isn't installed by default (because it's big).  So this test is going to get skipped a lot.  It might work properly in the places we need it to - we could check the CI/CD pipelines to make sure they're installing the extras.  But even if we do that, this test becomes somewhat fragile in that it needs `numpy` and can effectively "fail silently" if numpy isn't there.  Whereas `pillow` / PIL is installed by default.

So I'd really rather we get this functionality without relying on numpy.